README.md

```diff
@@ -2,6 +2,34 @@
 
 This repository contains GCP connectors for Apache Flink.
 
+## Examples
+
+You can find multiple examples that use GCP connectors on `flink-examples-gcp`.
+
+These are:
+- Google Cloud Storage Load Generator
+- Google Cloud Storage to Google Cloud Storage with Table API
+- Google Cloud Storage to Google Cloud Storage Unbounded WordCount
```

Comment:
let's be remove the unbounded here and let it imply that streaming is the default and then for the bounded one let's remove the word "Bounded" and just label it as "(Batch)" instead